Header header
float32 count_ch1
float32 count_ch2
float32 count_ch3
float32 count_ch4

